src/elements/Image/Image.tsx

```diff
@@ -56,8 +69,7 @@ export const Image: FunctionComponent<Props> = ({
                     id={`image-${file.uuid}`}
                     href={href}
                     className="prezly-slate-image__link"
-                    target="_blank"
-                    rel="noopener noreferrer"
+                    {...(isNewTab ? NEW_TAB_ATTRIBUTES : [])}
```

Comment:
Shouldn't the right-hand part be empty object instead? 🤔

Reply:
Good catch! Thank you!